packages/react/src/infoTag/MdInfoTag.tsx

```diff
@@ -59,7 +64,7 @@ const MdInfoTag: React.FC<MdInfoTagProps & React.ButtonHTMLAttributes<HTMLButton
     }
   };
   return onClick ? (
-    <button type="button" onClick={onClick} className={classNames}>
```

Comment:
Kanskje vi burde beholde denne? Litt dumt hvis man må spesifisere at disse komponente ikke skal være type="submit" med type="button". Flere steder i md komp har vi lagt på denne for å unngå at de er type="submit" som default.